LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 LibreTexts
```

Comment:
```suggestion
Copyright (c) 2018-2020 metalc authors
```

Reply:
@celine168 Can you also make an `AUTHORS` file in each repo and list first name and last name of the contributors?

Reply:
Make the copyright look like above, using the years the repo was created to the current year.

Reply:
Got it!